src/popup/router/components/TransactionItem.vue

```diff
@@ -10,7 +10,9 @@
       </div>
       <div class="holder">
         <span class="url" @click="visitTipUrl">{{ tipUrl }}</span>
-        <span class="seeTransaction" @click="seeTx(transactionData.hash)"><img :src="eye"/></span>
+        <span class="seeTransaction" @click="seeTx(transactionData.hash)">
+          <img src="../../../icons/eye.png" />
```

Comment:
Shouldn't this also be imported as the rest of the resources?

Reply:
Vue-loader handles src attribute of img well, I don't see reasons to don't use that 🤷‍♀️, it is just shorter